cpp/src/prims/count_if_v.cuh

```diff
@@ -35,9 +35,9 @@ namespace detail {
 
 template <typename vertex_t, typename VertexValueInputIterator, typename VertexOp>
 struct count_if_call_v_op_t {
-  vertex_t local_vertex_partition_range_first{};
-  VertexValueInputIterator vertex_value_input_first{};
-  VertexOp v_op{};
+  vertex_t local_vertex_partition_range_first;
```

Comment:
I think we can default initializing vertex_t (which is either int32_t or int64_t) or can this be also problematic?